 active-rfcs/0001-issue-helper.md

```diff
@@ -0,0 +1,106 @@
+- Feature Name: Issue Helper
+- Start Date: 2020-02-20
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+We'll use GitHub Issue [contact_links](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) to create issues, and in contact_links we'll put an external web page called issue-helper to help us create issues in a standard format.
+
+# Motivation
+
+In the past, TiDB's code repository used GitHub Issues to create bug reports, but because GitHub Issues use markdown to organize and write the report content. This caused our bug reports to be missing the basic information needed to reproduce the bug, making it impossible for our developers to reproduce the bug.
```

Comment:
```suggestion
In the past, TiDB's code repository used GitHub Issues to create bug reports, but because GitHub Issues use markdown to organize and write the report content. This form is simple but not structured, and not easy to use, so information transmission is inefficient.
```

Whether or not it is said that it will be easier to explain the problem we want to solve. After being structured, automation can be done, and it is not limited to the reproduction of certain bugs